src/app/content/components/Page/PageComponent.tsx

```diff
@@ -195,6 +212,9 @@ export default class PageComponent extends Component<PagePropTypes> {
 
     lazyResources.addScrollHandler();
 
+    if (this.props.inAssigned) {
```

Comment:
instead of short circuiting on `inAssigned`, you should check for the target attribute in the click handler. but actually i just tested and it seems like links with `target="blank"` on them work as expected already

Reply:
Should checking for a target attribute be added to [here](https://github.com/openstax/rex-web/blob/main/src/app/content/components/Page/contentLinkHandler.ts#L121)?

Reply:
Handling of links with target blank seemed to work already when I tested it, are you having problems with it?

Reply:
I've been testing it by clicking the figure 1 link. When I added the `target="_blank"` attribute, it still opened in the same tab, navigating to the Rex page.
I see that the "Simplify an expression" link properly opens in another tab, but that aborts [here](https://github.com/openstax/rex-web/blob/main/src/app/content/components/Page/contentLinkHandler.ts#L134) whereas the figure links have a `reference` and so don't abort there.

Reply:
Ok so yea I think you were right about handling the target attribute here https://github.com/openstax/rex-web/pull/1919#discussion_r1176853470


But I think there is also a different bug here having to do with figure links on the same page, we can address that in a separate change if you want

Reply:
I've put in a fix for both of these issues. Clicking a figure link works, but awkwardly: it looks like the component re-mounts, so it scrolls to the top of the page and re-displays toasts, then scrolls down to the figure, which winds up partly obscured by the TopBar.

Reply:
Oh, but it broke a test, so maybe not a correct fix.